main.go

```diff
@@ -282,6 +283,11 @@ func doMain() {
 	r.POST("/", func(context *gin.Context) {
 		payload, err := github.ValidatePayload(context.Request, conf.githubSecret)
 		if err != nil {
+			var mbErr *http.MaxBytesError
+			if errors.As(err, &mbErr) {
```

Comment:
please double check, this is a pointer to a pointer now passed to As -- this is what you want?